### DISCUSSION TITLE
Awesome Blogs On DSA & Open Source

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [EddieHubCommunity/support#1122] [Show and tell] [answered]
Hello I have shifted to Dev.to as my primary blogging platform do checkout my profile : https://dev.to/kaiwalyakoparkar

**Comment by kaiwalyakoparkar** [comment]
https://dev.to/kaiwalyakoparkar/steps-to-take-before-starting-a-new-open-source-project-23c6

**Comment by kaiwalyakoparkar** [comment]
https://dev.to/kaiwalyakoparkar/hackathons-all-you-need-to-know-right-now-h6e

**Comment by kaiwalyakoparkar** [comment]
https://dev.to/kaiwalyakoparkar/15-vs-code-extension-you-need-as-a-developer-19pg

**Comment by kaiwalyakoparkar** [comment]
This is a blog on helping you understand jargons like frontend, api, backend, server, database etc : https://dev.to/kaiwalyakoparkar/front-end-v-s-back-end-2fkk

**Comment by kaiwalyakoparkar** [comment]
This is the blog on installing Ubuntu as the main OS in the easiest way. I personally followed this step
https://dev.to/kaiwalyakoparkar/installing-ubuntu-a-step-by-step-guide-2b29

**Comment by kaiwalyakoparkar** [comment]
https://dev.to/kaiwalyakoparkar/roadmap-to-full-stack-development-2631

**Comment by kaiwalyakoparkar** [comment]
Blog by @felixfaisal on How to implement linked list in rust
https://dev.to/felixfaisal/implementing-linked-list-in-rust-3and

**Comment by kaiwalyakoparkar** [comment]
I have accumulated almost 40 terminal commands that can ease you life a bit while using your computer : https://dev.to/kaiwalyakoparkar/linux-commands-a-complete-handbook-1a7b

**Comment by aminoxix** [comment]
A very first blog of mine 🚀 
All about Open Source, quick start, & Opportunities : https://amino19.medium.com/open-source-opportunities-20b23db9e5a

Nice to see you here actively, @kaiwalyakoparkar

**Reply by kaiwalyakoparkar** [reply]
Hope you become more active too :)

**Reply by aminoxix** [reply]
Lets see, for community only I start this blogging activity to get updated with a friend as you as well <3

**Reply by kaiwalyakoparkar** [reply]
Glad to have you :blush:

**Comment by kaiwalyakoparkar** [comment, marked as answer]
All about EddieHub. If you are beginner you should definitely give it a look
https://dev.to/kaiwalyakoparkar/accelerate-your-learning-and-grow-your-network-with-eddiehub-1icn

**Comment by kaiwalyakoparkar** [comment]
Setting up your project more easily using Linters, Prettier, Huskey
https://dev.to/kaiwalyakoparkar/upgrade-your-project-with-linters-prettier-husky-cd1

**Comment by kaiwalyakoparkar** [comment]
Creating your own discord bot using dicord js: https://dev.to/kaiwalyakoparkar/creating-your-own-discord-bot-using-js-2ep0

**Comment by kaiwalyakoparkar** [comment]
Hey folks, I completed 1 week as an intern in DoK Community so I shared my 1-week experience here:  https://kaiwalyakoparkar.hashnode.dev/week-1-joining-dok-as-a-community-management-intern and I will be documenting my journey every week (6 months) in form of series. Make sure you signup for the newsletter. :raised_hands::relieved:

**Reply by ashwinexe** [reply]
That's an amazing experience Kaiwalya ^_^

**Reply by Sam-Varghese** [reply]
Awesome 🤩🤩this beginner would surely take guidance from you once he gets to know more about these big-big events 😊

Heartiest congrats for achieving so much!! 👍👍⚡

**Reply by kaiwalyakoparkar** [reply]
Thank you @ashwinexe and @Sam-Varghese. We have twitter space today at 9PM IST. Do join it :smiley:

**Reply by ashwinexe** [reply]
> Thank you @ashwinexe and @Sam-Varghese. We have twitter space today at 9PM IST. Do join it 😃

Public Speaking session on Twitter? Sounds good.

**Reply by kaiwalyakoparkar** [reply]
@ashwinexe No no it's not public speaking session. It's the first Twitter space of DoK.

**Reply by ashwinexe** [reply]
Oh, haha! awesome I'll be there.

**Comment by kaiwalyakoparkar** [comment]
Hey folks, Many or you would be working with API's building and deploying them. But do you know the structured way of building or designing API. For that I have written a blog on **RESTful API Design (In 5 easy steps)**. I am leaning more about API's so I would like to listen your thought on this :) 
https://kaiwalyakoparkar.hashnode.dev/restful-api-design-building-structured-apis-in-5-easy-steps

**Comment by kaiwalyakoparkar** [comment]
Hey everyone, I completed my 2nd week as a Data on Kubernetes Community intern and these are my learning and experience so far. You can read my experience in the form of a Blog here:
https://kaiwalyakoparkar.hashnode.dev/week-2-new-project-and-new-task
